core/citrus-spring/src/main/java/com/consol/citrus/common/SpringXmlTestLoaderConfiguration.java

```diff
@@ -1,24 +1,22 @@
 package com.consol.citrus.common;
 
-import com.consol.citrus.config.handler.CitrusTestCaseNamespaceHandler;
-
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
 /**
- * Xml test loader annotation used to configure parsers in {@link CitrusTestCaseNamespaceHandler}
- * 
+ * Xml test loader annotation used to configure parsers in {@link com.consol.citrus.config.handler.CitrusTestCaseNamespaceHandler}
+ *
  * @author T. Schlathoelter
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
-public @interface XmlTestLoaderConfiguration {
+public @interface SpringXmlTestLoaderConfiguration {
```

Comment:
I think we use this somewhere, renaming would potentially be breaking.

Reply:
Yep, this is breaking. Can you rename to the new annotation usage in your projects? The name should be aligned with the test loader name. We can deprecate the old annotation for some releases and remove then if you want.